model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py

```diff
@@ -17,33 +17,33 @@
 from inspect import signature
 
 from model_compression_toolkit.core import common
-from model_compression_toolkit.core.common.quantization.node_quantization_config import NodeWeightsQuantizationConfig, \
-    NodeActivationQuantizationConfig, BaseNodeQuantizationConfig
 from model_compression_toolkit.core.common.target_platform import QuantizationMethod
 
 from model_compression_toolkit.quantizers_infrastructure.common.base_inferable_quantizer import BaseInferableQuantizer, \
     QuantizationTarget
+from model_compression_toolkit.quantizers_infrastructure.common.base_trainable_quantizer_config import BaseQuantizerConfig, \
+    TrainableQuantizerActivationConfig, TrainableQuantizerWeightsConfig
 
 
 class BaseTrainableQuantizer(BaseInferableQuantizer):
     def __init__(self,
-                 quantization_config: BaseNodeQuantizationConfig,
+                 quantization_config: BaseQuantizerConfig,
```

Comment:
if removing the BaseQuantizerConfig, you can hint either of the 2 other configs:
`quantization_config:TrainableQuantizerActivationConfig | TrainableQuantizerWeightsConfig`